LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -551,6 +551,9 @@ - (void)calculateOffsetGap {
         _verticalOffset = -5;
         _passcodeButtonGap = _verticalGap;
     }
+    if (_displayedAsLockScreen && _isUsingNavBar) {
+        _verticalOffset += 44;
```

Comment:
can you explain why we need this?

Reply:
When on lock screen with nav bar (log out button), the label will be covered by the nav bar.
Need to move down to avoid that.